app/views/external_activities/_form.html.haml

```diff
@@ -68,6 +68,12 @@
     = field_set_tag 'Student Report Enabled' do
       = f.check_box :student_report_enabled
       = t('authoring.student_report_enabled_description')
+    = field_set_tag 'Saves Student Data' do
+      - if current_user.has_role? 'admin','manager'
```

Comment:
This is already inside of a block: `if @external_activity.new_record? || policy(@external_activity).edit_settings?` That will allow material admins to change this setting.  A material admin is a project admin for a project that this material is associated with.  So I think you can remove the check above.